ports/srpc/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "srpc",
-  "version-semver": "0.9.3",
+  "version-semver": "0.9.7",
```

```suggestion
  "version-semver": "0.9.7",
  "license": "Apache-2.0",
```
Please add the license. :)